src/responder/common/responder_common.c

```diff
@@ -219,11 +219,11 @@ errno_t csv_string_to_uid_array(TALLOC_CTX *mem_ctx, const char *csv_string,
         goto done;
     }
 
-    if (allow_sss_loop) {
-        ret = unsetenv("_SSS_LOOPS");
-        if (ret != EOK) {
-            DEBUG(SSSDBG_OP_FAILURE, "Failed to unset _SSS_LOOPS, getpwnam "
-                                      "might not find sssd users.\n");
+    if (!allow_sss_loop) {
```

Comment:
Now that SSS_LOOPS is not set for callers of `csv_string_to_uid_array`, I think it makes sense to rename `allow_sss_loop` to `set_sss_loops`, or `prevent_sss_loops` and keep it false by default, only call setenv when it is true (calling function arguments would need to be inverted)

Reply:
I renamed the argument ``set_sss_loops`` as you proposed.